conf/schema.xml

```diff
@@ -107,6 +110,16 @@
       </analyzer>
     </fieldType>
 
+    <!-- for autocomplete suggester -->
+    <fieldType class="solr.TextField" name="textSuggest" positionIncrementGap="100">
+       <analyzer>
+          <tokenizer class="solr.KeywordTokenizerFactory"/>
+          <filter class="solr.StandardFilterFactory"/>
+          <filter class="solr.LowerCaseFilterFactory"/>
+          <filter class="solr.RemoveDuplicatesTokenFilterFactory"/>
+       </analyzer>
+    </fieldType>
+
```

Comment:
Do we need a `textSpell` fieldType block as well? 

https://github.com/sul-dlss/earthworks/blob/master/config/solr_configs/schema.xml#L128